packages/ai/tests/conftest.py

```
@@ -13,7 +13,19 @@
 from unittest.mock import MagicMock
 
 
+class MockIJson(dict):
+    """Mock IJson class that supports isinstance checks and dict methods."""
+
+    pass
```

Ruff (0.15.7)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)